netbsd/NetBSDProcessTable.c

```diff
@@ -151,7 +151,7 @@ static double getpcpu(const NetBSDMachine* nhost, const struct kinfo_proc2* kp)
 }
 
 static void NetBSDProcessTable_scanProcs(NetBSDProcessTable* this) {
-   const Machine* host = this->super.host;
+   const Machine* host = this->super.super.host;
```

Comment:
As `this->super` is handed to `ProcessTable_getProcess` below it might be worth splitting this assignment and extract `super` into it's own variable as is already done with most other platform implementations of `*ProcessTable_scanProcs`.

Reply:
I had a look at this and simplified it based on what I saw in the `darwin` port.

- https://github.com/htop-dev/htop/blob/main/darwin/DarwinProcessTable.c#L72

I created a follow up PR against my branch to show the changes.

@BenBE : Would you like me to commit that also into this PR and merge both the commits in this single PR?

Reply:
Go ahead and cherry-pick. Changes look easy enough.

Reply:
Done.